apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.ejs

```diff
@@ -176,6 +193,31 @@
                   <% }); %>
                   <% } %>
                 </td>
+                <td class="text-right">
+                  <div class="dropdown js-question-actions">
+                    <button type="button" class="btn btn-secondary btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                      Action <span class="caret"></span>
+                    </button>
+
+                    <div class="dropdown-menu">
+                      <% if (authz_data.has_course_instance_permission_edit) { %>
+                        <button
+                          id="row<%= iRow %>PopoverBreakButton"
+                          class="dropdown-item js-mark-variant-as-broken"
+                          data-toggle="popover"
+                          data-trigger="click"
+                          data-container="body"
+                          data-html="true"
+                          data-placement="auto"
+                          title="Confirm mark as broken"
+                          data-content="<%= include('popoverBreakVariants', {iRow: iRow, question: question}) %>"
+                        >
+                          Mark as broken
```

Comment:
Same naming comment as for instance questions. We are not marking the assessment question as broken, so I'd prefer something like "Generate new variants" or "Reset question variants".

Reply:
Language changed on this page in 7e0b69c.